provision/debian/control

```diff
@@ -2,7 +2,8 @@ Source: acc-provision
 Section: python
 Priority: optional
 Maintainer: Cisco Systems, Inc. <dhami@noironetworks.com>
-Build-Depends: debhelper (>= 9), python-setuptools (>= 3.3), python-all (>= 2.7)
+X-Python-Version: >= 2.6
```

Comment:
I don't think this cherry-pick is recent (notice the two cherry-pick lines in the commit message). Better to just do a fresh cherry-pick in a new branch and post a new PR.